docs/userguide/monitoring.rst

```diff
@@ -99,12 +151,12 @@ Workflow Summary
 
 The workflow summary page captures the run level details of a workflow, including start and end times
 as well as task summary statistics. The workflow summary section is followed by the *App Summary* that lists
-the various apps and invocation count for each. 
+the various apps and invocation count for each.
 
 .. image:: ../images/mon_workflow_summary.png
 
 
-The workflow summary also presents three different views of the workflow:
+The workflow summary also presents three or four different views of the workflow (the number depends on whether file provenance is enabled and files were used in the workflow):
```

Comment:
could change "three" to "several" and then you don't need the whole if-statement-in-brackets

Reply:
ok